### DISCUSSION TITLE
Добавить возможность создавать пользователей веб панели MikoPBX

### DISCUSSION THREAD

**Discussion by cblvnche** [mikopbx/Core#286] [Ideas (Идеи)]
Будет полезно иметь возможность создавать пользователей в веб-панели MikoPBX, чтобы можно было управлять привилегиями и уровнями доступа к тем или иным функциям, модулям и страницам. Сейчас приходится предоставлять полный доступ супервайзеру, а это не безопасно

**Comment by boffart** [comment]
https://github.com/mikopbx/Core/discussions/197
аналогичное обсуждение, ответ тот же. 
планы есть, но не ближайшие
